subprojects/gradle-guides-plugin/src/main/java/org/gradle/docs/internal/StringUtils.java

```diff
@@ -41,7 +42,45 @@ public static String uncapitalize(String s) {
      * @return transformed string
      */
     public static String toTitleCase(String s) {
-        return Arrays.stream(GUtil.toWords(s).split(" ")).map(StringUtils::capitalize).collect(Collectors.joining(" "));
+        return Arrays.stream(toWords(s).split(" ")).map(StringUtils::capitalize).collect(Collectors.joining(" "));
```

Comment:
❓ I'm wondering if [`String.split`](https://docs.groovy-lang.org/3.0.11/html/gapi/index.html?org/codehaus/groovy/runtime/StringGroovyMethods.html) is enough here?